gwentifyHandler.py

```diff
@@ -82,7 +82,17 @@ def getCardJson(html):
     content = cardArticle.find('div', class_="entry-content")
 
     dataMap["name"] = name.strip()
-    dataMap["imageUrl"] = imageUrl.strip()
+    dataMap["variations"] = []
+
+    # Only one variation in gwentify, but the game can have many variations of a card
+    variation = {}
+    variation["availability"] = "BaseSet" # Currently all cards are from the base set.
+    art = {}
+    art["fullsizeImageUrl"] = imageUrl.strip()
+
+    variation["art"] = art
+
+    dataMap["variations"].append(variation)
```

Comment:
Introduced the variations from [here](https://gist.github.com/jamieadkins95/b3f23961d5260e26b2798fa6240a9f0e).